test/suite/wttest.py

```diff
@@ -203,7 +203,7 @@ class WiredTigerTestCase(unittest.TestCase):
     @staticmethod
     def globalSetup(preserveFiles = False, removeAtStart = True, useTimestamp = False,
                     gdbSub = False, lldbSub = False, verbose = 1, builddir = None, dirarg = None,
-                    longtest = False, ignoreStdout = False, seedw = 0, seedz = 0, hookmgr = None):
+                    longtest = False, zstdtest = False, ignoreStdout = False, seedw = 0, seedz = 0, hookmgr = None):
```

Comment:
Should this new `zstdtest` argument be set in the function call in `test/suite/run.py`?
```
    wttest.WiredTigerTestCase.globalSetup(preserve, removeAtStart, timestamp, gdbSub, lldbSub,
                                          verbose, wt_builddir, dirarg, longtest,
                                          ignoreStdout, seedw, seedz, hookmgr)
```

Reply:
Thanks Luke, I am going to add this argument now and then run another patch build.